README.md

```diff
@@ -31,7 +31,7 @@
 </p>
 
 <p align="center">
-  <a href="https://trendshift.io/repositories/2570" target="_blank"><img src="https://trendshift.io/api/badge/repositories/2570" alt="Trendshift Badge for formbricks/formbricks" style="width: 250px; height: 55px;" width="250" height="55"/></a>
+  <a href="https://trendshift.io/repositories/2570" target="_blank"><img src="https://trendshift.io/api/badge/repositories/2570" alt="Trendshift Badge for formbricks/formbricks" style="width: 250px; height="55px;" width="250" height="55"/></a>
```

Comment:
is this intentional or a prettier issue?

Reply:
Was done mistakenly, reverted it back